src/react-apps/applications/runtime/src/actions/formFillerActions/actions/fetchAttachments/index.ts

```diff
@@ -0,0 +1,38 @@
+import { Action } from 'redux';
+import * as ActionTypes from '../../formFillerActionTypes';
+
+export interface IFetchAttachmentsAction extends Action {
```

Comment:
Not really a need to extend Action interface with nothing.
Make the fetchAttachments just return an Action.

Reply:
Fixed.